common/src/main/java/org/jboss/hal/testsuite/page/config/UndertowHostPage.java

```diff
@@ -0,0 +1,17 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+
+public class UndertowHostPage extends UndertowHTTPPage {
+
+    public ConfigFragment switchToReferenceToFilterSubTab() {
```

Comment:
The name doesn't need to correspond exactly to the subtitle as that subtitle can be changed. Suggesting to rename it to `switchToFilterReferencesSubTab` or add javadoc.